classes/import-export.php

```diff
@@ -476,11 +476,12 @@ public function import_csv() {
 													$term_id = $term_array['term_id'];
 												}
 
-												// This is just a failsafe. It also gives us access to vars the WP API created rather than from the CSV.
-												if ( ! is_wp_error( $term_id ) && $term = get_term( (int) $term_id, $taxonomy ) && ! is_wp_error( $term ) ) {
-													// Associate (last var appends term to rather than replaces existing terms).
-													wp_set_object_terms( $id, $term->name, $taxonomy, true );
-													unset( $term );
+												// This is just a failsafe. It also gives us access to vars the WP API created rather than from the CSV
+												if ( ! is_wp_error( $term_id ) && ( $term = get_term( (int) $term_id, $taxonomy ) ) && ! is_wp_error( $term ) ) {
+														// Associate (last var appends term to rather than replaces existing terms)
+														wp_set_object_terms( $id, $term->name, $taxonomy, true );
+														unset( $term );
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found

<!-- Comment by [SideCI](https://sideci.com) -->